### PR TITLE
feat: add Dependabot support for Github Actions

### PR DESCRIPTION
**Pull request by thecristen**
I was going to add a PR to update a reference to a particular outdated action used in a particular workflow, but I figure I can also enable this to have Dependabot create the PRs for me. Any thoughts? :)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#about-the-dependabotyml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "mbta/ctd-infrastructure"
```
Comment (krisrjohnson21) [inline .github/dependabot.yml]: This makes sense to me with one caveat--I'm not sure @mbta/ctd-infrastructure is the right group to review dependabot PRs in the actions repo. Any thoughts, Ian or Paul?

**Review by krisrjohnson21 — COMMENTED** [review]
One open thought from me, would love to hear additional Infra feedback

File: .github/dependabot.yml (hunk 1 above)
Comment (paulswartz) [reply .github/dependabot.yml]: I would agree with that. Maybe we should have an `@mbta/tech-leadership` group to round-robin between the leads?

Comment (ianwestcott) [reply .github/dependabot.yml]: I just created @mbta/tid-tech-leads for this purpose. @thecristen can you update the PR?

Comment (thecristen) [reply .github/dependabot.yml]: Done

**Review by krisrjohnson21 — APPROVED** [review]
🧼

**Pull request merged**